src/Nest/Cluster/NodesStats/NodeStats.cs

```diff
@@ -571,4 +574,74 @@ public class HttpStats
 		[DataMember(Name = "total_opened")]
 		public long TotalOpened { get; internal set; }
 	}
+
+	[DataContract]
+	public class IndexingPressureStats
+	{
+		[DataMember(Name = "memory")]
+		public MemoryStats Memory { get; internal set; }
+
+		[DataContract]
+		public class MemoryStats
```

Comment:
While I think inner classes are the best choice stylistically once we shift from hand rolling these to generating these it will be hard to know when inner classes are preferred. 

I hate saying it but [smurf types](https://blog.codinghorror.com/new-programming-jargon/) are better here

![smurf typing](https://media.giphy.com/media/APbcWKvZf5ka7hU19j/giphy-downsized-large.gif)

Reply:
That's a new term for me 😄

Reply:
I've moved out to non-nested classes @Mpdreamz. If this change is what you expected, hopefully we're good to merge.